### PR TITLE
Introduce first iteration of glossary

### PR DESCRIPTION
**Pull request by leopic**
@jfsagasti and I had talked about this a few months ago.

I was waiting a to add a little bit more substance but I figured something is better than nothing, so here is my proposal for a basic glossary, we can keep expanding on it as we go.

### CODE REVIEW

File: Glosario.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Glosario
+
+- **strong reference**: referencia fuerte
+- **gesture recognizer**: reconocedor de gestos
+- **debug**: depurar
+- **key-value coding**: key-value coding
+- **game center**: game center
+- **hard-coded**: alambrado
```
Comment (jfsagasti) [inline Glosario.md]: Hi @leopic! Thanks for the glossary. I think we can use hard-coded as it is. "Alambrado" is something I've never read before and I think readers from Spain will not get that easily. Also, almost every programmer knows what a _hardcoded_ value is :)

Comment (leopic) [reply Glosario.md]: Hahaha you are right, I removed it from the glossary :)

**Comment by leopic** [comment]
PR has been updated

**Comment by leopic** [comment]
No problem, thank you for your reviews.

That's your cue @mattt

**Comment by leopic** [comment]
I don't have a strong opinion on it, I tend to favor having the docs close (or in the same place) as the code, in this case, so having the glossary in the same place makes sense to me.

I'm fine either way :)

**Comment by jfsagasti** [comment]
A wiki sounds great!

**Comment by leopic** [comment]
Hey folks, I was under the impression that only maintainers could edit the Wiki, but I just created the first page [over here](https://github.com/NSHipster/articles-es/wiki/Glosario).

Closing this one for now, thanks!

**Pull request closed without merging**